sphinxdocs/src/sphinx_bzl/bzl.py

```diff
@@ -498,7 +498,40 @@ def run(self) -> list[docutils_nodes.Node]:
         self.env.ref_context["bzl:file"] = file_label
         self.env.ref_context["bzl:object_id_stack"] = []
         self.env.ref_context["bzl:doc_id_stack"] = []
-        return []
+
+        _, _, basename = file_label.partition(":")
```

Comment:
Is `file_label` guaranteed to have a colon?

Reply:
Yes. Even for packages, the notation is `//foo:BUILD.bazel` to denote the file that a BUILD-file defined thing comes from.